tests/tests_unit/test_api/test_vision_extract.py

```diff
@@ -46,12 +48,24 @@ def mock_get_response_body_ok() -> Dict[str, Any]:
         "items": [
             {
                 "fileId": 1,
-                "predictions": [{"text": "testing", "assetIds": [1, 2, 3], "confidence": 0.9}],
-            },
+                "predictions": {
+                    "assetTagPredictions": [
+                        {
+                            "assetRef": {"id": 1},
+                            "confidence": 0.9,
+                            "text": "testing",
+                            "textRegion": {
+                                "xMin": 0.1,
+                                "yMin": 0.2,
+                                "xMax": 0.3,
+                                "yMax": 0.4,
+                            },
+                        }
+                    ]
+                },
+            }
         ],
-        "useCache": True,
-        "partialMatch": True,
-        "assetSubtreeIds": [39468345],
+        "parameters": {"assetTagDetectionParameters": {"assetSubtreeIds": [], "partialMatch": False, "threshold": 0.4}},
```

Comment:
If predictions is correct, maybe it should be updated in the response samples here: https://docs.cognite.com/api/v1/#tag/Vision/operation/postVisionExtract

Reply:
This is a sample response from the _GET_ endpoint (not POST), which I believe is compatible with what we have in the API docs (does not include all optional fields)

Reply:
Yep, looked at the wrong one